pyqtorch/core/utils.py

```diff
@@ -22,6 +22,17 @@
 ABC_ARRAY: NDArray = np.array(list(ABC))
 
 
+IMAT = torch.eye(2, dtype=torch.cdouble)
+XMAT = torch.tensor([[0, 1], [1, 0]], dtype=torch.cdouble)
+YMAT = torch.tensor([[0, -1j], [1j, 0]], dtype=torch.cdouble)
+ZMAT = torch.tensor([[1, 0], [0, -1]], dtype=torch.cdouble)
+SMAT = torch.tensor([[1, 0], [0, 1j]], dtype=torch.cdouble)
+TMAT = torch.tensor([[1, 0], [0, torch.exp(torch.tensor(1j) * torch.pi / 4)]], dtype=torch.cdouble)
+SWAPMAT = torch.tensor([[1, 0, 0, 0], [0, 0, 1, 0], [0, 1, 0, 0], [0, 0, 0, 1]], dtype=torch.cdouble)
+
+OPERATIONS_DICT = {"I" : IMAT, "RX" : XMAT, "RY" : YMAT, "RZ" : ZMAT, "S" : SMAT, "T": TMAT, "SWAP" : SWAPMAT}
```

Comment:
I don't understand why you add the operations in utils

Reply:
I am also not sure about it. I just noticed that they are redefined several times so I wanted to avoid repetition